Sources/GDSAnalytics/Events/EventName.swift

```diff
@@ -2,4 +2,5 @@ public enum EventName: String {
     case navigation
     case formResponse = "form_response"
     case popup
+    case screenView = "screen_view"
```

Comment:
should have caught this before, but not sure if we need this, believe firebase already takes care of providing the event name for screen view, see here https://github.com/govuk-one-login/mobile-ios-logging/blob/main/Sources/GAnalytics/GAnalytics.swift#L45

This line has a constant called `AnalyticsEventScreenView` which has the string `screen_view`

Reply:
Fair! Will remove this for now and see if it works without.